patches/chrome-installer-mac-sign_versioned_dir.sh.in.patch

```diff
@@ -34,9 +34,6 @@ index a9db32c8ec17629f745eff38a6c732124a21f63e..b2ac9db824bdf769474beb0477b2b06b
 -                      "com.google.Chrome.helper"
```

Comment:
I thought we left the chrome helper process alone and only renamed the main exe?

Reply:
Would have to look into it, this is what we've been testing with at least (https://github.com/brave/brave-core/pull/112/files#diff-4072779e50936048e252d687f263df84R34).